### ISSUE TITLE
 Social.ReportScore not working

### ISSUE THREAD

**Issue by JrPro1** [playgameservices/play-games-plugin-for-unity#2041]
Hello, 

I am not able to report the score to the leaderboard. 
```
long score = long.Parse("6");
if (Social.localUser.authenticated)
{
                Social.ReportScore(score, leaderboard, (bool success) =>
                {  
                    if (success)
                    {
                        Debug.Log("Update Score Success");
                    }
                    else
                    {
                        Debug.Log("Update Score Fail");
                    }
                });
}
```

and I get Update Score Success in the log. 

please help.

**Comment by usescrt** [comment]
Same problem here.

**Comment by hipopasran** [comment]
Same problem, it work, but leaderbords dont update score. I use plugin vers 0.9.41 and Unity 2017.2.0f3.

**Comment by Solid-Metal** [comment]
same here , not only that, leaderboard also crash

**Comment by hipopasran** [comment]
@Solid-Metal, @usescrt , @JrPro1 
I downgrade the version of the Unity to 2017.1.2f1 and it worked, try, maybe it will help, I have not found another solution so far.

**Comment by usescrt** [comment]
I downgrade the version Unity 2017.1.2f1 still same
I give you some more detail

1. sign in its work.
2. achievement its work.
3. if leaderboard empty, its work and open.

But I posted score , then not open again leaderboard.
I've been stuck here for a week, please help.

**Comment by vijayvjc8** [comment]
Yes, I'm facing this problem from a month, taught I was doing some wrong code, please do something I'm stuck here from month, I use current build of gpgs and unity 2017.2.of3........someone if you know how to solve this pls help us

**Comment by xk0fe** [comment]
> 1. sign in its work.
> 2. achievement its work.
> 3. if leaderboard empty, its work and open.

Same on Unity 2019.1.0f2

**Comment by felipesflisboa** [comment]
#2542 solved this issue for me.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
